mz_os.c

```diff
@@ -33,7 +33,9 @@ int32_t mz_path_combine(char *path, const char *join, int32_t max_path) {
         path[max_path - 1] = 0;
     } else {
         mz_path_append_slash(path, max_path, MZ_PATH_SLASH_PLATFORM);
-        strncat(path, join, max_path - path_len);
+        path_len = (int32_t)strlen(path);
```

Comment:
Can't we just do `patch_len += 1;` here?

Reply:
We could do that. In the case where mz_path_append_slash does not append a slash (because it's already present), it means we will truncate the filename one character shorter than we need to.

I can respin that way if you think performance is more important than filling max_path.

Reply:
I see. It's probably good for now the way it is.